### PR TITLE
Static pages

### PR DESCRIPTION
**Pull request by makoto**
- Remove about and team page (we don't need it until we get rid of the current landing page)
- Change copyright to company
- Make contact email address visible
- Add faq, privacy, and terms page

NOTE: I haven't renumbered terms and policy. Will wait for Ram's input before renumbering.

### CODE REVIEW

File: src/layout/Footer.js [hunk 1]
```diff
@@ -68,21 +68,23 @@ class Footer extends Component {
       <FooterContainer>
         <FooterInner>
           <H2>Get in touch</H2>
-          <a href="mailto:hello@kickback.events">hello@kickback.events</a>
+          <p>
+            <a style={{color:'white'}}href="mailto:hello@kickback.events">hello@kickback.events</a>
```
Comment (hiddentao) [inline src/layout/Footer.js]: Need space before href. Also, why aren't we using styled-components for this? See other files for examples of how to style elements, or e.g. `Row` element below.

File: src/routes/Privacy.js [hunk 2]
```diff
@@ -0,0 +1,125 @@
+import React, { Component } from 'react'
+
+class Privacy extends Component {
+  render() {
+    return (
+      <div className="privacy">
```
Comment (hiddentao) [inline src/routes/Privacy.js]: Should not be using string class names. Please used styled-components instead.

**Comment by makoto** [comment]
@hiddentao when I tried to use styled component like below, the default color of `a` was overriding my setting and could not make it work. I reverted my change for now so that @jefflau can do it properly 

```
const Contact = styled('p')`
  a{
    color:'white';
  }
```

File: src/routes/Privacy.js (hunk 2 above)
Comment (makoto) [reply src/routes/Privacy.js]: I actually copied and pasted the one Jeff originally created which had these classname (it's used in other places as well) so I just followed the existing convention. Can we make styled component bits out of scope of this review, as my focus is to put the content in, not about styling?

**Pull request merged**